hack/check-file-length.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+
+filename=$1
+if [ -z "$filename" ]; then
+    echo >&2 $0 expects a filename
+    exit 1
+fi
+
+token=lint_maximum_file_length
+max=$(grep "$token" "$filename" | awk '{print $3}')
```

Comment:
Couldn't this be a single target max length? Is setting it per-file necessary?
`max=2000`

Reply:
> Is setting it per-file necessary?

IMHO, yes. I would like to see `test/utils.go` shrink to 200 lines to down to zero. 90% percent of the content should sit in a properly-named package. On the other hand, we have several huge files which are vendored, or auto-generated, or where I otherwise don't have a clear path to reduce. I'd like to cap this specific file and start reducing it. I hope one day we can do the same for `pkg/virt-handler/vm.go`, but I do not think we are ready yet.

Reply:
fair point to bring up vendor and auto-generated.
Perhaps then, we limit it to scanning only those files or directories that we do have more control over, perhaps only excluding those which cannot be reduced at this time.
This way we can use a single limit, and not have to manage a token per file.

Reply:
I'm reverting to a simplistic static proposal.